x509-identity-mgmt/docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+version: "3.7"
```

Comment:
create a folder examples and add this file (example: https://github.com/dojot/dojot/tree/epic-routing/util/microservice-sdk/js/examples)

Reply:
For the time being I just made an example. I would like to have a scenario closer to the real one to exemplify a scenario of multiple instances, but for now, I think this example is cool.

Preview:
https://github.com/thiagodpf/dojot/tree/x509-identity-mgmt/x509-identity-mgmt/examples/simple